src/cheatsheet.html

```diff
@@ -23,9 +23,9 @@ <h2 class="page-header">Every Font Awesome {{ site.fontawesome.version }} Icon,
     {% assign sorted_icons = icons | expand_aliases | sort_by:'class' %}
 
     {% for icon in sorted_icons %}
-    <div class="col-md-4 col-sm-6 col-lg-3">
```

Comment:
Is there any way to leave these classes in for screen rendering and as importantly transparency? Can we target and override the print styling with the print media-query-based rules only?